File: Leetcode/Leet1365.java
//https://leetcode.com/problems/how-many-numbers-are-smaller-than-the-current-number/
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int[] arr = new int[nums.length];
        for(int i = 0; i< nums.length; i++){
            for(int j = 0; j<nums.length; j++){
                if((i!=j) && (nums[j] < nums[i])){
                    arr[i] += 1;
                }
            }
        } return arr;
    }
}

File: Leetcode/Leet1365.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        arr = []
        for i in range(len(nums)):
            count = 0
            for j in range(len(nums)):
                if j!= i and nums[j] < nums[i]:
                    count += 1
            arr.append(count)
        return arr